wordpress-mobile/WordPress-Android
# Issue/5285 title content capitalization

### Fix
Capitalize the first letter of sentences in the title and content fields of the editor as described in https://github.com/wordpress-mobile/WordPress-Android/issues/5285.

### Test
1. Go to ***Sites*** tab.
2. Tap ***Create*** floating button.
3. Tap title field.
4. Enter multiple sentences.
5. Notice first letter of each sentence is capitalized.
6. Tap content field.
7. Enter multiple sentences.
8. Notice first letter of each sentence is capitalized.
